src/sysusers/sysusers.c

```diff
@@ -117,7 +117,7 @@ static int load_user_database(void) {
                         return q;
                 }
 
-                if (q <= 0 && k <= 0)
+                if (k <= 0 && q <= 0)
```

Comment:
It seems that `load_group_database` should be changed similarly as well.

Reply:
Updated.